packages/medusa/src/services/line-item.ts

```diff
@@ -334,7 +428,7 @@ class LineItemService extends TransactionBaseService {
     )
   }
 
-  protected async generateLineItem(
+  protected async generateLineItem_(
```

Comment:
comment: Breaking change

Reply:
so we should use another name for the new method and keep this one?